apitests/gulpfile.js

```diff
@@ -38,7 +38,9 @@ gulp.task('build', 'Runs build:clean, tsd and ts in sequence', function (callbac
 });
 
 gulp.task('test-jasmine', 'Starts jasmine to run tests', (done) => {
-    return gulp.src(['dist/**/*.js'])
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
```

Comment:
actually I think it's better if its in the helpers. Cause if we delete this gulp task or use a different one for running tests then it's going to fail.